spec/models/computacenter/chromebook_details_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Computacenter::ChromebookDetails do
+  subject(:service) { described_class }
+
+  describe '.to_csv' do
+    let(:chromebook_details) { create_list(:preorder_information, 5, :needs_chromebooks) }
+    # let(:expected_headers) do
+    #   [
+    #     'Responsible Body URN',
+    #     'Responsible Body Name',
+    #     'School Name',
+    #     'School URN',
+    #     'Google Domain',
+    #     'Valid Recovery Off Domain Email Address',
+    #     'Date',
+    #     'Time',
+    #   ]
+    # end
```

Comment:
delete this block of commented code please

Reply:
Good spot!